### PR TITLE
Update docker image version of golang and alpine

### PR DESCRIPTION
**Pull request by viveksyngh**
This change is to update the Dockerfiles to use the latest version of
docker images which has multi arch support.

All golang images are updated to golang:1.9.2 and alpine images are updated to
 alpine:3.7

Signed-off-by: Vivek Singh <vivekkmr45@yahoo.in>

## Motivation and Context

- [ ] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by iyovcheva** [comment]
@viveksyngh Can you please fill in the template: how has this been tested and the checklist?
Is there an issue raised for this change?

**Comment by alexellis** [comment]
Please check with master as I believe I made this change yesterday after testing.

**Comment by viveksyngh** [comment]
@alexellis In this pull request, I have updated for `Dockerfile.arm64` and `Dockefile.non-multi` images version.

**Comment by rgee0** [comment]
If we are moving to multi-arch base images is there any opportunity to consolidate any of the Dockerfiles?

**Comment by viveksyngh** [comment]
I think `Dockerfile.arm64` and `Dockerfile.armhf` can be consolidated into one.

**Comment by alexellis** [comment]
@rgee0 there are a couple of things that need work.. multi-arch binaries for this tool and an installation script that pulls the platform-specific version.

https://github.com/alexellis/license-check

Otherwise we have to comment this line out in the various Dockerfiles.

**Comment by alexellis** [comment]
https://github.com/alexellis/license-check/issues/1

**Comment by alexellis** [comment]
I think we can delete the file named "Dockerfile.non-multi" though

File: Dockerfile.non-multi [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM golang:1.7.5
+FROM golang:1.9.2
```
Comment (alexellis) [inline Dockerfile.non-multi]: Let's delete `Dockerfile.non-multi`

File: Dockerfile.arm64 [hunk 2]
```diff
@@ -1,5 +1,4 @@
-FROM alexellis2/golang:1.9-arm64 as build
-#FROM golang:1.7.5
+FROM golang:1.9.2 as build
```
Comment (alexellis) [inline Dockerfile.arm64]: Should work but can we verify on Scaleway?

Comment (viveksyngh) [reply Dockerfile.arm64]: I think I have verified on scaleway. I can confirm once again.

File: Dockerfile.non-multi (hunk 1 above)
Comment (viveksyngh) [reply Dockerfile.non-multi]: I will delete this.

**Comment by viveksyngh** [comment]
I have deleted the `Dockefiel.non-multi` and tested the `arm64` file on scaleway. It is working. 

```
root@scw-fa8150:~/faas-netes# make build-arm64
docker build -t functions/faas-netesd:latest-arm64 . -f Dockerfile.arm64
Sending build context to Docker daemon  23.82MB
Step 1/14 : FROM golang:1.9.2 as build
 ---> e3e365ec7aa8
Step 2/14 : ENV GOPATH=/go/
 ---> Using cache
 ---> 9231c85192d9
Step 3/14 : RUN mkdir -p /go/src/github.com/openfaas/faas-netes/
 ---> Using cache
 ---> 3dddcd29ea7a
Step 4/14 : WORKDIR /go/src/github.com/openfaas/faas-netes
 ---> Using cache
 ---> 1d7ce9fb9a14
Step 5/14 : COPY . .
 ---> df524925073e
Step 6/14 : RUN gofmt -l -d $(find . -type f -name '*.go' -not -path "./vendor/*")   && VERSION=$(git describe --all --exact-match `git rev-parse HEAD` | grep tags | sed 's/tags\///')   && GIT_COMMIT=$(git rev-list -1 HEAD)   && CGO_ENABLED=0 GOOS=linux go build --ldflags "-s -w     -X github.com/openfaas/faas-netes/version.GitCommit=${GIT_COMMIT}    -X github.com/openfaas/faas-netes/version.Version=${VERSION}"     -a -installsuffix cgo -o faas-netes .
 ---> Running in 7e0c11608cfb
Removing intermediate container 7e0c11608cfb
 ---> dcb56d5e13e4
Step 7/14 : FROM alpine:3.7
 ---> f6df4608fa98
Step 8/14 : RUN apk --no-cache add ca-certificates
 ---> Using cache
 ---> b8af7b1fb8f3
Step 9/14 : WORKDIR /root/
 ---> Using cache
 ---> e00be93014ae
Step 10/14 : EXPOSE 8080
 ---> Using cache
 ---> 8973eabb540f
Step 11/14 : ENV http_proxy      ""
 ---> Using cache
 ---> 75b453568e17
Step 12/14 : ENV https_proxy     ""
 ---> Using cache
 ---> f6f80dac58cc
Step 13/14 : COPY --from=0 /go/src/github.com/openfaas/faas-netes/faas-netes    .
 ---> b7e6c2853278
Step 14/14 : CMD ["./faas-netes"]
 ---> Running in 362399927ad3
Removing intermediate container 362399927ad3
 ---> 0808a3eaf8d7
Successfully built 0808a3eaf8d7
Successfully tagged functions/faas-netesd:latest-arm64
```

**Comment by alexellis** [comment]
Thank you for doing the multi-arch testing/builds. I'll take a final look and hopefully just merge it at that point.

**Comment by alexellis** [comment]
LGTM

**Pull request merged**